velox/exec/tests/TaskTest.cpp

```diff
@@ -1314,4 +1314,43 @@ TEST_F(TaskTest, driverCreationMemoryAllocationCheck) {
     }
   }
 }
+
+TEST_F(TaskTest, spillDirectoryLifecycleManagement) {
+  // Marks the spill directory as not already created and ensures that the Task
+  // handles creating it on first use and eventually deleting it on destruction.
+  auto data = makeRowVector({
```

Comment:
nit: consider to use fuzzer input row

Reply:
as discussed offline, keeping this unchanged as adding a fuzzer input row will mean we included those files but there is no additional benefit of doing so